org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/conv30_40/resources30_40/MedicationRequest30_40.java

```diff
@@ -62,17 +76,79 @@ public static org.hl7.fhir.dstu3.model.MedicationRequest convertMedicationReques
       tgt.addDetectedIssue(Reference30_40.convertReference(t));
     for (org.hl7.fhir.r4.model.Reference t : src.getEventHistory())
       tgt.addEventHistory(Reference30_40.convertReference(t));
+
```

Comment:
Same issues as above. If there's added R3 -> R4 functionality for a field, it should be included in the test.